gym_ai2thor/config_files/config_example.json

```diff
@@ -16,6 +16,10 @@
     "scene_id": "FloorPlan28",
     "grayscale": true,
     "resolution": [128, 128],
+    "cameraY": 0.0,
+    "gridSize": 0.1,
+    "incremental_rotation": false,
```

Comment:
Can we change the name of incremental_rotation to "rotation_step" and set it as default to the original value, i.e. 90?

Reply:
The implementation of 90 degree rotates `RotateLeft` is different than the implementation of 89 degree rotates `Rotate` absolute degrees, so I'd rather not

Reply:
Could have unintended behaviour and I like the default and common RotateLeft/Right